### PR TITLE
aa-*: add pages

### PR DESCRIPTION
**Pull request by miki164**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/linux/aa-complain.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# aa-complain
+
+> Set an AppArmor policy to complain mode.
+> More information: <https://gitlab.com/apparmor/apparmor/-/wikis/manpage_aa-complain.8>.
+
+- Set policy to complain mode:
+
+`sudo aa-complain {(path/to/profile})`
```
Comment (blueskyson) [inline pages/linux/aa-complain.md:8]:
```suggestion
`sudo aa-complain {{path/to/profile}}`
```

File: pages/linux/aa-complain.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Set policies to complain mode:
+
+`sudo aa-complain --dir {(path/to/profiles)}`
```
Comment (blueskyson) [inline pages/linux/aa-complain.md:12]:
```suggestion
`sudo aa-complain --dir {{path/to/profiles}}`
```

File: pages/linux/aa-disable.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# aa-disable
+
+> Disable AppArmor security policy.
+> More information: <https://gitlab.com/apparmor/apparmor/-/wikis/manpage_aa-disable.8>.
+
+- Disable profile:
+
+`sudo aa-disable {(path/to/profile})`
```
Comment (blueskyson) [inline pages/linux/aa-disable.md:8]:
```suggestion
`sudo aa-disable {{path/to/profile}}`
```

File: pages/linux/aa-disable.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+- Disable profiles:
+
+`sudo aa-disable --dir {(path/to/profiles)}`
```
Comment (blueskyson) [inline pages/linux/aa-disable.md:12]:
```suggestion
`sudo aa-disable --dir {{path/to/profiles}}`
```

File: pages/linux/aa-enforce.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# aa-enforce
+
+> Set an AppArmor profile to enforce mode.
+> More information: <https://gitlab.com/apparmor/apparmor/-/wikis/manpage_aa-enforce.8>.
+
+- Enable profile:
+
+`sudo aa-enforce {(path/to/profile})`
```
Comment (blueskyson) [inline pages/linux/aa-enforce.md:8]:
```suggestion
`sudo aa-enforce {{path/to/profile}}`
```

File: pages/linux/aa-enforce.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+- Enable profiles:
+
+`sudo aa-enforce --dir {(path/to/profiles)}`
```
Comment (blueskyson) [inline pages/linux/aa-enforce.md:12]:
```suggestion
`sudo aa-enforce --dir {{path/to/profiles}}`
```

**Review by blueskyson — APPROVED** [review]

File: pages/linux/aa-complain.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# aa-complain
+
+> Set an AppArmor policy to complain mode.
```
Comment (kbdharun) [inline pages/linux/aa-complain.md:3]:
```suggestion
> Set an AppArmor policy to complain mode.
> See also: `aa-disable`, `aa-enforce`, `aa-status`.
```

File: pages/linux/aa-disable.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# aa-disable
+
+> Disable AppArmor security policy.
```
Comment (kbdharun) [inline pages/linux/aa-disable.md:3]:
```suggestion
> Disable AppArmor security policy.
> See also: `aa-complain`, `aa-enforce`, `aa-status`.
```

File: pages/linux/aa-enforce.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+# aa-enforce
+
+> Set an AppArmor profile to enforce mode.
```
Comment (kbdharun) [inline pages/linux/aa-enforce.md:3]:
```suggestion
> Set an AppArmor profile to enforce mode.
> See also: `aa-complain`, `aa-disable`, `aa-status`.
```

File: pages/linux/aa-status.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# aa-status
+
+> List currently loaded AppArmor modules.
```
Comment (kbdharun) [inline pages/linux/aa-status.md:3]:
```suggestion
> List currently loaded AppArmor modules.
> See also: `aa-complain`, `aa-disable`, `aa-enforce`.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**